templates/solochain/node/Cargo.toml

```diff
@@ -0,0 +1,92 @@
+[package]
+name = "solochain-template-node"
+version = "4.0.0-dev"
+description = "A solochain node template built with Substrate, part of Polkadot Sdk."
+authors.workspace = true
+homepage.workspace = true
+repository.workspace = true
+edition.workspace = true
+license = "MIT-0"
+publish = false
+
+[lints]
+workspace = true
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[[bin]]
+name = "solochain-template-node"
+
+[dependencies]
+clap = { version = "4.5.1", features = ["derive"] }
+futures = { version = "0.3.21", features = ["thread-pool"] }
+serde_json = { workspace = true, default-features = true }
+jsonrpsee = { version = "0.22", features = ["server"] }
+
+# substrate client
+sc-cli = { path = "../../../substrate/client/cli" }
+sp-core = { path = "../../../substrate/primitives/core" }
+sc-executor = { path = "../../../substrate/client/executor" }
+sc-network = { path = "../../../substrate/client/network" }
+sc-service = { path = "../../../substrate/client/service" }
+sc-telemetry = { path = "../../../substrate/client/telemetry" }
+sc-transaction-pool = { path = "../../../substrate/client/transaction-pool" }
+sc-transaction-pool-api = { path = "../../../substrate/client/transaction-pool/api" }
+sc-offchain = { path = "../../../substrate/client/offchain" }
+sc-consensus-aura = { path = "../../../substrate/client/consensus/aura" }
+sp-consensus-aura = { path = "../../../substrate/primitives/consensus/aura" }
+sc-consensus = { path = "../../../substrate/client/consensus/common" }
+sc-consensus-grandpa = { path = "../../../substrate/client/consensus/grandpa" }
+sp-consensus-grandpa = { path = "../../../substrate/primitives/consensus/grandpa" }
+sc-client-api = { path = "../../../substrate/client/api" }
+sc-rpc-api = { path = "../../../substrate/client/rpc-api" }
+sc-basic-authorship = { path = "../../../substrate/client/basic-authorship" }
+
+# substrate primitives
+sp-runtime = { path = "../../../substrate/primitives/runtime" }
+sp-io = { path = "../../../substrate/primitives/io" }
+sp-timestamp = { path = "../../../substrate/primitives/timestamp" }
+sp-inherents = { path = "../../../substrate/primitives/inherents" }
+sp-keyring = { path = "../../../substrate/primitives/keyring" }
+sp-api = { path = "../../../substrate/primitives/api" }
+sp-blockchain = { path = "../../../substrate/primitives/blockchain" }
+sp-block-builder = { path = "../../../substrate/primitives/block-builder" }
+
+# frame and pallets
```

Comment:
Did you group the dependencies or were they grouped already?  
Personally find stuff faster in an alphabetically sorted list, but not a strong opinion 🙈

Reply:
Yea, not sure how its configured currently. CI should fail if its wrong.